package.json

```diff
@@ -64,6 +64,7 @@
     "test": "sh scripts/lint.sh && sh scripts/tests.sh",
     "test:server": "sh scripts/tests.sh",
     "lint": "sh scripts/lint.sh",
+    "prewatch:client": "cd client && npm install",
```

Comment:
It's our standard process, to ensure client deps are available. When client apps will become pure client-side apps and we avoid this messy double project repos, all of this shit will be dropped 😉